build-uboot-standalone.sh

```
@@ -0,0 +1,166 @@
+
+# ARM64 cross-compiler - try Yocto SDK first, then system
+YOCTO_BUILD="${PWD}/build"
+SDK_PATH="${YOCTO_BUILD}/tmp/deploy/sdk"
+
+ENV_SETUP=""
+if [ -d "/opt/poky" ]; then
+    ENV_SETUP=$(find /opt/poky -maxdepth 1 -name 'environment-setup-*' -print -quit 2>/dev/null)
+fi
+if [ -n "$ENV_SETUP" ]; then
+    echo "Using installed Yocto SDK from /opt/poky"
+    source "$ENV_SETUP"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)